### ISSUE TITLE
TypeError: Cannot read property 'object' of undefined

### ISSUE THREAD

**Issue by the-simian** [finnfiddle/number-picture#3]
The 'hello world' does not run

TypeError: Cannot read property 'object' of undefined
./node_modules/number-picture/cjs/mixins/AnimatedElement.js
E:/Users/Simian/Desktop/test-project/node_modules/number-picture/cjs/mixins/AnimatedElement.js:409

```
AnimatedElement.propTypes = {
  datum: _react.PropTypes.object, ///THIS LINE FAILS
  datumAccessor: _react.PropTypes.oneOfType([_react.PropTypes.func]),
  enterDatum: _react.PropTypes.oneOfType([_react.PropTypes.object, _react.PropTypes.func]),
  exitDatum: _react.PropTypes.oneOfType([_react.PropTypes.object, _react.PropTypes.func]),
  enterEase: _react.PropTypes.oneOf(EASE_TYPES),
  updateEase: _react.PropTypes.oneOf(EASE_TYPES),
  exitEase: _react.PropTypes.oneOf(EASE_TYPES),
  enterDuration: _react.PropTypes.number,
  updateDuration: _react.PropTypes.number,
  exitDuration: _react.PropTypes.number,
  propsToCheckForChanges: _react.PropTypes.arrayOf(_react.PropTypes.string),
  datumPropsToTween: _react.PropTypes.arrayOf(_react.PropTypes.string)
};
```
I am using => "react": "^16.1.1",

**Comment by finnfiddle** [comment]
a bit delayed here but it doesnt yet support react@16. Will add this soon

**Comment by finnfiddle** [comment]
this is fixed in v0.5.0 and is publish on npm. 66792b579fca26b6a87e94fdbda7e21ef76c7cef
